src/test/app/WasmHostFuncImpl_test.cpp

```diff
@@ -2010,6 +2096,85 @@ struct WasmHostFuncImpl_test : public beast::unit_test::suite
         BEAST_EXPECT(result.value() == msg.size() + sizeof(num));
     }
 
+    void
+    testTraceAccount()
+    {
+        testcase("traceAccount");
+        using namespace test::jtx;
+
+        Env env{*this};
+        OpenView ov{*env.current()};
+        ApplyContext ac = createApplyContext(env, ov);
+
+        auto const dummyEscrow =
+            keylet::escrow(env.master, env.seq(env.master));
+        WasmHostFunctionsImpl hfs(ac, dummyEscrow);
+
+        std::string msg = "trace account";
+        // Valid account
+        {
+            auto const result = hfs.traceAccount(msg, env.master.id());
+            BEAST_EXPECT(result.has_value());
+            BEAST_EXPECT(
+                result.value() ==
+                msg.size() + toBase58(env.master.id()).size());
+        }
+
+        // Invalid (zero) account
+        {
+            auto const result = hfs.traceAccount(msg, xrpAccount());
+            BEAST_EXPECT(!result.has_value());
```

Comment:
Same here and below
` if BEAST_EXPECT...`

Reply:
[37cad83](https://github.com/XRPLF/rippled/pull/5674/commits/37cad8334675c5e1008761a3fed89202294adfb4)